client/my-sites/subscribers/hooks/use-unsubscribe-modal.ts

```diff
@@ -29,7 +29,7 @@ const useUnsubscribeModal = (
 	const onConfirmModal = ( action: UnsubscribeActionType, subscriber?: Subscriber ) => {
 		if ( action === UnsubscribeActionType.Manage ) {
 			recordRemoveModal( true, 'manage_button_clicked' );
-			window.open( getEarnPaymentsPageUrl( selectedSiteSlug ), '_blank' );
+			navigate( `/earn/supporters/${ selectedSiteSlug }` );
```

Comment:
What about adding a
```
const getEarnSupportersPageUrl = ( siteSlug: string | null ) => `/earn/subscribers/${ siteSlug ?? '' }`;

```
to keep the same paradigm?

Reply:
https://github.com/Automattic/wp-calypso/pull/85423/commits/efa09863adc55a1ea0fe8b4e83383c185e937827

Reply:
This does not answer my review at all...